### DISCUSSION TITLE
Formula/portmidi.rb uses obsolete Makefile.osx

### DISCUSSION THREAD

**Discussion by rbdannenberg** [Homebrew/discussions#3313] [Tap maintenance and Homebrew development]
I recently discovered Makefile.osx in PortMidi sources, but I believe it is a "legacy" file that should be removed. There has been recent work on CMake files including a lot of additions to support installations on all platforms. Also, there seem to be problems with Makefile.osx, but no tests, no specification, and no maintainers, so my proposal was to simply remove it. Then it was pointed out that Homebrew uses it. Do you want to be responsible for Makefile.osx contents? Do you want to write a specification? Would it be easier to use cmake? I don't want to pull the rug out from under anyone, but I don't want to pretend that Makefile.osx is up-to-date or even usable.

**Comment by carlocab** [comment]
That’s likely just an oversight. We should switch it to using CMake.
